recipe-server/client/control/components/action_fields/HeartbeatFields.js

```diff
@@ -5,7 +5,7 @@ import { ControlField } from 'control/components/Fields';
 /**
  * Form fields for the show-heartbeat action.
  */
-export default function HeartbeatFields({ fields }) {
+export default function HeartbeatFields({ fields = {} }) {
```

Comment:
This should probably be called `arguments` instead of `fields`, since the value that gets passed in is the arguments object. But that's actually a refactoring that should happen in a different PR, since the form selector is also named `recipeFields`.

Whenever we get to that refactoring, we might consider putting the selector inside this file instead of putting it inside the RecipeForm and passing the value downwards, as it makes it more obvious where this value is coming from.